debian/cocaine-runtime.conf

```diff
@@ -22,7 +25,7 @@
         }
     },
     "paths": {
-        "plugins": "/usr/lib/cocaine",
```

Comment:
No, reading from both `/usr/` and `/usr/local` seems pointless. Moreover on the development environment it can be dangerous, where you have deb packets installed in `/usr/lib` simultaneously with the development libraries in `/usr/local`.

It's not **required** to change default configuration. So revert.